pkg/controller/servicebindingrequest/servicebindingrequest_controller.go

```diff
@@ -130,7 +130,7 @@ func (r *ReconcileServiceBinding) Reconcile(request reconcile.Request) (reconcil
 }
 
 // newPodForCR returns a busybox pod with the same name/namespace as the cr
-func newPodForCR(cr *appsv1alpha1.ServiceBinding) *corev1.Pod {
+func newPodForCR(cr *appsv1alpha1.ServiceBindingRequest) *corev1.Pod {
```

Comment:
Please remove this? :)

Reply:
That's coming from the boiler plate generated by the SDK. I will remove it in a separate PR :-)

Reply:
Done. 4b58bcce25668d1b5737c60d1c2e76db20fc2f5f